From 3f6e7285f93ec2304553dfc8c6bbbfbe05566b04 Mon Sep 17 00:00:00 2001
From: David Robillard <d@drobilla.net>
Date: Fri, 12 Mar 2021 23:36:30 -0500
Subject: [PATCH] Make the type of patch:wildcard more precise

It is more or less meaningless to have an explicit type of rdfs:Resource, and
some tools don't like it, since everything (including classes and properties)
is a Resource.  Since this is an individual, owl:Thing is a better type.
---
 lv2/patch/patch.ttl | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/lv2/patch/patch.ttl b/lv2/patch/patch.ttl
index 0fb2012f..c7f6d769 100644
--- a/lv2/patch/patch.ttl
+++ b/lv2/patch/patch.ttl
@@ -236,7 +236,7 @@ patch:value
 	rdfs:comment "The value of a property in a patch:Set message." .
 
 patch:wildcard
-	a rdfs:Resource ;
+	a owl:Thing ;
 	rdfs:label "wildcard" ;
 	rdfs:comment "A wildcard that matches any resource." .
 
